examples/jdbc2-complex-type-queries/jdbc2_complex_type_queries.description

```diff
@@ -0,0 +1,6 @@
+// This example demonstrates how to use the JDBC client with complex data types such as arrays,
+// binary, blobs, clob, and date/time fields.
+// Please note that, the relevant database driver jar
+//should be placed under `BALLERINA_HOME/bre/lib` folder. However this sample is based on
```

Comment:
```suggestion
//should be placed under the `BALLERINA_HOME/bre/lib` folder. However, this sample is based on an
```

Reply:
Need a space after //. Shall we fix everywhere?